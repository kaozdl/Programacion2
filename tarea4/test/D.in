construir_cnj (30,abcd) (2,bacd) (3,ccc) (1,d) (5,e) (1,f) (7,g) (1,h) (4,i) (2,j) (3,k) (1,l) (5,m) (1,n) (7,o) .
ops_cnj (20,xx) .
excluir abcd bacd ccc .
excluir d f e g h j i k m l n o .
Fin
