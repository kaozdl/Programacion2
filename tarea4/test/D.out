1>2>1
0
3>4>Vacio.
5>Fin.
